static/js/lib/video.js

```diff
@@ -55,3 +56,16 @@ export const videoHasError = R.compose(
   ),
   R.prop('status')
 );
+
+export const saveToDropbox = (video: Video) => {
+  const sourceVideos = video.videofile_set.filter((videofile:VideoFile) => (videofile.encoding === 'original'));
+  if (sourceVideos && sourceVideos.length > 0) {
+    const extension = sourceVideos[0].s3_object_key.split('.').pop();
+    if (window.Dropbox) {
+      window.Dropbox.save(
+        `${SETTINGS.cloudfront_base_url}${encodeURI(sourceVideos[0].s3_object_key)}`,
+        sanitize(video.title + (video.title.endsWith(extension) ? '' : `.${extension}`))
```

Comment:
2 things: 
1. can you make a util function in `urls.js` that takes a `VideoFile` and creates that url? I know it's a simple string concatenation, but it would be nice to avoid constructing these video urls in a bunch of different places
1. there's a lot going on here. can you create local vars for the URL and filename, set them to the two lines above respectively, then pass those into Dropbox.save? it would help clarify exactly what we're giving to this 3rd party library function

Reply:
OK